dubbo-config/dubbo-config-api/src/main/java/com/alibaba/dubbo/config/ProtocolConfig.java

```diff
@@ -454,6 +456,14 @@ public void setExtension(String extension) {
         this.extension = extension;
     }
 
+    public Boolean getDecodeInIo() {

```

Comment:
Add @Parameter(key = "decode.in.io")

Reply:
thank you for your repley.there is another group who use the similiar way,and it seems they  dont have to change the src code.I think their way is better.